miasm2/jitter/arch/JitCore.c

```diff
@@ -1,8 +1,127 @@
 #include <Python.h>
```

Comment:
As `JitCore` is totally arch independent, it could be placed in the parent directory, like `vm_mngr`, `queue`, `Jitcoretcc`, ...